desktop/core/src/desktop/js/reactComponents/FileChooser/FileSystemList/FileSystemList.tsx

```diff
@@ -0,0 +1,35 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { Menu } from 'antd';
+
+import './FileSystemList.scss';
+import { FileSystem } from '../types';
+
+interface FileSystemProps {
+  fileSystems: Array<FileSystem>;
+}
+
+const FileSystemList: React.FC<FileSystemProps> = ({ fileSystems }) => {
```

Comment:
Are you planning on adding more functionality to this component? Because now I don't see the need for having this as its own component since it is basically just wrapping the ant Menu.

Reply:
The only other functionality would be sending the home directory path to the files listing api to display the files in the table. I could move the menu back to the FileChooserModal file. Should I continue with that?